monitoring/lifecycle/dashboard.py

```diff
@@ -1,257 +1,764 @@
-            f'sum(increase(s3_lifecycle_kafka_publish_success_total{{{label}}}[$__rate_interval]))',
-        ),
-        kafka_messages_time_series(
-            f'{topic} Failed Messages',
-            f'sum(increase(s3_lifecycle_kafka_publish_error_total{{{label}}}[$__rate_interval]))',
+messages_published_by_op = TimeSeries(
+    title='Message published',
+    dataSource="${DS_PROMETHEUS}",
+    fillOpacity=5,
+    lineInterpolation='smooth',
+    unit=UNITS.REQUESTS_PER_SEC,
+    targets=[
+        Target(
+            expr='sum(rate(' + Metrics.KAFKA_PUBLISH_SUCCESS() + ')) by(op)',
+            legendFormat="{{ op }}",
         ),
-    ]
+    ],
+)
 
+# Basically a wrapper for KAFKA_PUBLISH_ERROR' rate, but filling the gaps with 0,
+# while keeping the same labels.
+kafka_publish_error_rate_expr = '\n'.join([
+    '(sum(rate(' + Metrics.KAFKA_PUBLISH_ERROR() + ')) by(op)',
+    '  or 0 * group(' + Metrics.KAFKA_PUBLISH_SUCCESS.raw() + ') by(op))',
+])
 
-up = Stat(
-    title="Up",
+failed_publish_by_op = TimeSeries(
+    title='Publish errors',
     dataSource="${DS_PROMETHEUS}",
-    reduceCalc="last",
-    noValue='0',
+    fillOpacity=5,
+    lineInterpolation='smooth',
+    unit=UNITS.REQUESTS_PER_SEC,
     targets=[
         Target(
-            expr='sum(up{namespace="${namespace}",job="${job_lifecycle_producer}"})',
-            legendFormat="Conductor",
-        ),
-        Target(
-            expr='sum(up{namespace="${namespace}",job="${job_lifecycle_bucket_processor}"})',
-            legendFormat="Bucket Processor",
+            expr=kafka_publish_error_rate_expr,
+            legendFormat="{{ op }}",
         ),
+    ],
+)
+
+publish_success_rate = Stat(
+    title='Publish success rate',
+    dataSource="${DS_PROMETHEUS}",
+    decimals=2,
+    format=UNITS.PERCENT_FORMAT,
+    graphMode='none',
+    orientation='horizontal',
+    reduceCalc='mean',
+    textMode='value_and_name',
+    targets=[
         Target(
-            expr='sum(up{namespace="${namespace}",job="${job_lifecycle_object_processor}"})',
-            legendFormat="Expiration Processor",
-        ),
+            expr='\n'.join([
+                '100 / (1 + ' + kafka_publish_error_rate_expr.replace('\n', '\n' +
+                '           '),
+                '           /',
+                '           (sum(rate(' + Metrics.KAFKA_PUBLISH_SUCCESS() + ')) by(op) > 0))'
+            ]),
+            legendFormat="{{ op }}",
+        )
     ],
+    thresholdType='percentage',
     thresholds=[
-        Threshold("green", 0, 0.0),
+        Threshold("green",  0, 0.0),
+        Threshold("yellow", 0, 0.1),
+        Threshold("orange", 0, 0.01),
+        Threshold("red",    1, 1.0),
     ],
 )
 
-lifecycle_batch = Stat(
-    title="Latest Batch Start Time",
+kafka_lag = TimeSeries(
+    title='Kafka Lag',
     dataSource="${DS_PROMETHEUS}",
-    reduceCalc="lastNotNull",
-    format='dateTimeAsLocal',
+    unit=UNITS.SHORT,
```

Comment:
same (up to you of course, it may be expected not to stylize these ones)
```suggestion
    unit=UNITS.SHORT,
    lineInterpolation='smooth',
```